website/docs/components/form/key-value-inputs/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Key Value Inputs
+description: >-
+  Rows of inputs that can be added and deleted by end users.
```

Comment:
```suggestion
  Rows of related inputs that can be individually added & deleted.
```

Reply:
Since it's not required to have multiple rows it could make sense to instead say:

"A row of related inputs that can be individually added & deleted."